met-web/src/components/engagement/form/EngagementFormTabs/EngagementTabsContext.tsx

```diff
@@ -90,16 +89,15 @@ export const EngagementTabsContext = createContext<EngagementTabsContextState>({
     setEngagementFormData: () => {
         throw new Error('setEngagementFormData is unimplemented');
     },
-    isNewEngagement: false,
-    setIsNewEngagement: () => {
-        throw new Error('setIsNewEngagement is unimplemented');
-    },
-    handleSaveEngagement: async () => {
-        console.warn('handleSaveEngagement is unimplemented');
+    handleSaveAndContinueEngagement: async () => {
+        console.warn('handleSaveAndContinueEngagement is unimplemented');
```

Comment:
Can I ask why this is necessary? Is this a concession made to allow for Typescript to work with useContext?

Reply:
Yes, just to satisfy the requirements of an interface. The actual implementation in done later.